prometheus-openstack-exporter

```diff
@@ -322,10 +423,35 @@ class Nova():
         l = [config['cloud'], 'disk']
         openstack_overcommit.labels(*l).set(config['openstack_allocation_ratio_disk'])
 
+    def gen_quota_stats(self):
+        cores = Gauge('nova_quota_cores',
+                      'Nova cores metric',
+                      ['cloud', 'tenant', 'type'], registry=self.registry)
+        fips = Gauge('nova_quota_floating_ips',
+                     'Nova floating IP addresses (number)',
+                     ['cloud', 'tenant', 'type'], registry=self.registry)
+        inst = Gauge('nova_quota_instances',
+                     'Nova instances (number)',
+                     ['cloud', 'tenant', 'type'], registry=self.registry)
+        ram = Gauge('nova_quota_ram_megabytes',
```

Comment:
We have `nova_resources_ram_mbs`, `hypervisor_memory_mbs_total` and similar. I think it's better to call this metric `nova_quota_ram_mbs` for consistency